### PR TITLE
docs: email contact point

### PR DESCRIPTION
**Pull request by tonypowa**
**What is this feature?**

- New doc for Email integration
- Plus a small edit on the Telegram integration doc

**Why do we need this feature?**

**Who is this feature for?**

**Which issue(s) does this PR fix?**:

Please check that:
- [X] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [X] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-email/
+description: Configure the Email integration to connect alerts generated by Grafana Alerting
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:3]: Configure email integration to send email notifications when your alerts are firing

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  - alerting
+  - email
+  - integration
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Email
+title: Configure Email for Alerting
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:15]:
```suggestion
title: Configure email for Alerting
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+weight: 300
+---
+
+## Configure Email for Alerting
+
+Use the Grafana Alerting - email integration to send email notifications when your alerts are firing. An email is sent when an alert fires and when an alert gets resolved.
+
+## Before you begin
+
+When utilizing Grafana OSS, enabling email notifications requires configuring [SMTP settings][smtp-settings] in the Grafana configuration settings. In contrast, for users of Grafana Cloud, SMTP configuration is not required.
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:25]: Add a note here 

Note:
This section is for Grafana OSS only. For Grafana Cloud, STMP configuration is not required.

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+1. **Open the Configuration File:**
+
+   Open the configuration file using a text editor.
+
+1. **Locate SMTP Settings Section.**
+
+   Search for the [SMTP settings section][smtp-settings] in the configuration file. It starts with `[smtp]`.
+
+1. **Configure SMTP Settings.**
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:41]: are these all screen elements? we only bold screen elements in this case

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+   - `enabled = true`: Enables SMTP.
+   - `host`: The hostname or IP address of your SMTP server, and the port number of your SMTP server (commonly 25, 465, or 587). Default is `localhost:25`.
+   - `user`: Your SMTP username (if authentication is required).
+   - `password`: Your SMTP password (if authentication is required).
+   - `from_address`: The email address from which Grafana notifications will be sent.
+   - `from_name`: The name associated with the from_address.
+   - `skip_verify = true`: Skip SSL/TLS certificate verification (useful for testing, but not recommended for production).
+
+1. **Save and Close the Configuration File.**
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:53]: same comment as above

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+1. Click **+ Add contact point**.
+1. Enter a contact point name.
+1. From the Integration list, select Email.
+1. Enter the email addresses you want to send notifications to.
+1. Click **Test** to check that your integration works.
+1. Click Save contact point.
+
+## Next steps
+
+To add the contact point and integration you created to your default **Notification policy**, complete the following steps.
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:79]:
```suggestion
To add the contact point and integration you created to your default notification policy, complete the following steps.
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md (hunk 4 above)
Comment (tonypowa) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:41]: nope they are not but i thought it increases readability.
i will "unbold" 😃  them

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md (hunk 5 above)
Comment (tonypowa) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:53]: ok here at least i will keep the code format so readers differentiate each element

Comment (tonypowa) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:53]: never mind, i see now you meant the  
1. **Save and Close the Configuration File.**


 not the above text

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Email
+title: Configure email for Alerting
+weight: 300
+---
+
+## Configure Email for Alerting
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:19]:
```suggestion
## Configure email for Alerting
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+Use the Grafana Alerting - email integration to send email notifications when your alerts are firing. An email is sent when an alert fires and when an alert gets resolved.
+
+## Before you begin
+
+When utilizing Grafana OSS, enabling email notifications requires configuring [SMTP settings][smtp-settings] in the Grafana configuration settings.
+
+{{<admonition type="note">}}
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:27]: can you put this note directly under "before you begin"?

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+This section is for Grafana OSS only. For Grafana Cloud SMTP configuration is not required.
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:28]:
```suggestion
This section is for Grafana OSS only. For Grafana Cloud, SMTP configuration is not required.
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+weight: 300
+---
+
+## Configure Email for Alerting
+
+Use the Grafana Alerting - email integration to send email notifications when your alerts are firing. An email is sent when an alert fires and when an alert gets resolved.
+
+## Before you begin
+
+When utilizing Grafana OSS, enabling email notifications requires configuring [SMTP settings][smtp-settings] in the Grafana configuration settings.
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:25]:
```suggestion
For Grafana OSS, you enable email notifications by first configuring [SMTP settings][smtp-settings] in the Grafana configuration settings.
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+   After restarting Grafana, test the email notification functionality by creating an email contact point.
+
+## Procedure
+
+To set up email integration, complete the following steps.
+
+1. Navigate to **Alerts & IRM** -> **Alerting** -> **Contact points**.
+1. Click **+ Add contact point**.
+1. Enter a contact point name.
+1. From the Integration list, select Email.
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:75]:
```suggestion
1. From the Integration list, select **Email**.
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+1. Enter the email addresses you want to send notifications to.
+1. Click **Test** to check that your integration works.
+1. Click Save contact point.
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md:78]:
```suggestion
1. Click **Save contact point**.
```

**Review by brendamuir — APPROVED** [review]

**Pull request merged**